### PR TITLE
Localization Helper

### PR DESCRIPTION
**Pull request by comasqw**
## About the PR

Resolving - https://github.com/crystallpunk-14/crystall-punk-14/issues/264
## Media

https://www.youtube.com/watch?v=5HfDjLzhjA4

### CODE REVIEW

**Review by TheShuEd — COMMENTED** [review]
Please eng error logs

File: Tools/_CP14/LocalizationHelper/main.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+
+def main():
+    config = read_config()
+    prototypes_path, localization_path = get_paths(config)
+    prototypes_dict, localization_dict = yml_parser.yml_parser(prototypes_path), ftl_parser.ftl_parser(localization_path)
+
+    entities_ftl = ""
+
+    def check_attrs(attr):
```
Comment (Deserty0) [inline Tools/_CP14/LocalizationHelper/main.py:25]: please move function from main

File: Tools/_CP14/LocalizationHelper/fluent/ftl_reader.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                                            }
+                else:
+                    if "desc" in line:
+                        attr = "desc"
+                    elif "suffix" in line:
+                        attr = "suffix"
+
+                    prototypes[last_prototype][attr] = line.split(" = ")[-1].strip()
+    except Exception as e:
+        print(f"Произошла ошибка во время чтения {path}, ошибка - {e}")
```
Comment (Deserty0) [inline Tools/_CP14/LocalizationHelper/fluent/ftl_reader.py:29]: en

File: Tools/_CP14/LocalizationHelper/yml_parser/yml_parser.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                with open(path, encoding="utf-8") as file:
+                    proto = ""
+                    for line in file.readlines():
+                        if "!type" in line:
+                            continue
+                        proto += line
+
+                    data = yaml.safe_load(proto)
+            except Exception as e:
+                print(f"Произошла ошибка во время обработки прототипа - {e}")
```
Comment (Deserty0) [inline Tools/_CP14/LocalizationHelper/yml_parser/yml_parser.py:38]: en

**Comment by comasqw** [comment]
Done, also added comments to the code

File: Tools/_CP14/LocalizationHelper/yml_parser/yml_parser.py [hunk 4]
```diff
@@ -16,6 +16,11 @@ def get_proto_attrs(prototypes: dict, prototype: dict) -> None:
 
 
 def yml_parser(path: str) -> dict:
+    """
+        The function gets the path, then with the help of the os library
+        goes through each file,checks that the file extension is "ftl",
```
[the remark this answers is not in the document]
Comment (comasqw) [reply Tools/_CP14/LocalizationHelper/yml_parser/yml_parser.py:21]: +

**Comment by comasqw** [comment]
> Don't merge it yet.
bug fixed

**Pull request merged**